README.md

```diff
@@ -76,7 +76,7 @@ For IPv6, returns true if is_local_ipv6 is true
 
 ### is_local_host
 
-Checks to see if a host name resolves to a local ip.
+Checks to see if a host name resolves to itself.
```

Comment:
`itself` is very very vague. Worse than 'a local ip' IMO. Maybe we make that clearer? Something like this is better: 'any IP address locally bound or looked up via STUN.'